File: 백준/Bronze/10214. Baseball/Baseball.py
game = int(input())

# 각각의 케이스마다 한 줄에 연세대가 이겼을 경우 "Yonsei", 고려대가 이겼을 경우 "Korea", 비겼을 경우 "Draw"를 출력한다.
for i in range(game) :
    
    yonseiScore = 0
    koreaScore = 0
    
    for i in range (9) :
        yonsei, korea = input().split()
        
        yonsei = int(yonsei)
        korea = int(korea)
        
        yonseiScore += yonsei
        koreaScore += korea
        
    if yonseiScore > koreaScore :
        print("Yonsei")
    elif yonseiScore < koreaScore :
        print("Korea")
    else :
        print("Draw")

File: 백준/Bronze/10214. Baseball/Baseball.java
import java.io.*;
import java.util.*;

public class Main {
	public static void main(String[] args) throws IOException {
		BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
		StringBuilder sb = new StringBuilder();
		StringTokenizer st; 
		
		int N = Integer.parseInt(br.readLine());
		
		for (int TC = 0; TC < N; TC++) {
			int Y = 0, K = 0;
			for (int i = 0; i < 9; i++) {
				st = new StringTokenizer(br.readLine());
				Y += Integer.parseInt(st.nextToken());
				K += Integer.parseInt(st.nextToken());
			}
			sb.append(Y>K?"Yonsei":(Y<K?"Korea":"Draw")).append("\n");
		}
		System.out.println(sb);
		br.close();
	}
}